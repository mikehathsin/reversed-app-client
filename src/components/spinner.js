import RBSpinner from "react-bootstrap/Spinner";

export const Spinner = () => {
  return (
    <RBSpinner animation="border" role="status">
      <span className="visually-hidden">Loading...</span>
    </RBSpinner>
  );
};
