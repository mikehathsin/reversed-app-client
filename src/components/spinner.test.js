import { render } from "@testing-library/react";

import { Spinner } from "./spinner";

describe("Spinner tests", () => {
  it("Should render correctly", () => {
    const component = render(<Spinner />);
    component.getByRole("status");
    expect(component.container).toMatchSnapshot();
  });
});
