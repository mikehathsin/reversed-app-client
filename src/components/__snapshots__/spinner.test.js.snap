// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner tests Should render correctly 1`] = `
<div>
  <div
    class="spinner-border"
    role="status"
  >
    <span
      class="visually-hidden"
    >
      Loading...
    </span>
  </div>
</div>
`;
